packages/cli/package.json

```diff
@@ -10,7 +10,7 @@
   ],
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "pnpm prebuild && tsc -p tsconfig.json",
+    "build": "pnpm prebuild && tsc -p tsconfig.json && cp -r src/commands/init/templates/{app*,github} dist/src/commands/init/templates",
```

Comment:
Tried to use `copyfiles` without luck. They don't support globs with recursion strangely.